tests/integration/session/test_timeout.py

```diff
@@ -23,9 +23,25 @@ def test_when_session_times_out_server_side_401_is_returned(self):
         time.sleep(5)
         self.get(self.last_url)
         self.assertStatusUnauthorised()
+        self.assertInBody("Your session has expired due to inactivity")
+
+    def test_alternate_401_page_is_displayed_when_no_cookie(self):
+        self.get("/session")
+        self.assertStatusUnauthorised()
+        self.assertInBody("Sorry there is a problem")
 
     def test_schema_defined_timeout_cant_be_higher_than_server(self):
         self.launchSurvey("test_timeout")
         time.sleep(4)
         self.get(self.last_url)
         self.assertStatusUnauthorised()
+        self.assertInBody("To help protect your information we have timed you out")
+
+    def test_submission_complete_timeout(self):
+        self.launchSurvey("test_timeout")
+        self.post()
+        self.post()
+        time.sleep(4)
+        self.get(self.last_url)
+        self.assertStatusUnauthorised()
+        self.assertInBody("Your census has been submitted")
```

Comment:
As the string `This page is no longer available` also appears on this page, it would be better to check that than census specific text.